cypress/support/pages/settingPageObjects.js

```diff
@@ -0,0 +1,69 @@
+import SignInPageObject from "./signIn.pageObject";
+
+class SettinsPageObject extends SignInPageObject {
+  url = '/settings';
+
+  loginUser() {
+    cy.login();
+  }
+
+  updateUsername(username) {
+    cy.getByDataCy('username-settings')
+      .clear()
+      .type(username);
+  }
+
+  updateBio(bio) {
+    cy.getByDataCy('bio-settings')
+      .type(bio);
```

Comment:
In the updateBio method, you might want to clear the previous bio before typing a new one, similar to the updateUsername and updateEmail methods.